docs/data/charts/tooltip/tooltip.md

```diff
@@ -90,141 +92,151 @@ It removes the header showing the x-axis value from the tooltip.
 />
 ```
 
-### Overriding content
+### Series
 
-To override tooltip content, provide a custom component to `slots.tooltip`.
-Some helper are provided, such as:
+To hide a series, the formatted value should be `null`.
+To display the series with a blank space, return an empty string.
 
-- `<ChartsTooltipContainer />` which provide a tooltip with built-in open and position management.
-- `useItemTooltip()` which provides all basic information associated to the current item.
-- `useAxesTooltip()` which provides all basic information associated to the current axes.
+## Style modification
 
-Here is the basic scheme to follow.
-Examples about helpers are provided in the composition section.
+The tooltip can be styled using CSS classes, similar to other elements.
+However, there is one caveat regarding using [portal](https://react.dev/reference/react-dom/createPortal).
 
-```jsx
-import { ChartsTooltipContainer } from '@mui/x-charts/ChartsTooltip';
+The tooltip renders as a child of the document's body element.
+From a DOM perspective, it's not inside the chart.
+So using the chart's `sx` prop as follow does not work.
+
+```tsx
+import { chartsTooltipClasses } from '@mui/x-charts';
+
+<LineChart
+  sx={{
+    [`& .${chartsTooltipClasses.root} .${chartsTooltipClasses.valueCell}`]: {
+      color: 'red',
+    },
+  }}
+/>;
+```
 
-function CustomItemTooltipContent() {
-  const tooltipData = useItemTooltip();
+To apply the same style as above, use the `sx` prop of the tooltip itself, thanks to `slotProps.tooltip`.
 
-  if (!tooltipData) { // No data to display
-    return null;
-  }
+{{"demo": "TooltipStyle.js"}}
 
-  return <div>{/** Your custom content **/}</div>;
-}
+Another option is to disable the portal by setting `slotProps.tooltip.disablePortal` to `true`.
+In that case, the tooltip renders as a child of the chart, and CSS rules apply as expected.
 
-<LineChart
-  slots={{ tooltip: CustomItemTooltip }}
-/>
+## Using a custom tooltip
 
-// With composition
+For advanced use cases, it can be necessary to create your own tooltip.
+You can replace the default tooltip in single component charts by using slots.
+
+```jsx
+<LineChart slots={{ tooltip: CustomItemTooltip }} />
+```
+
+With compositon, you can use your component inside the container.
+
+```jsx
 <ChartContainer>
   // ...
-  <ChartsTooltipContainer trigger="item">
-    <CustomItemTooltipContent />
-  </ChartsTooltipContainer>
+  <CustomItemTooltip />
 </ChartContainer>
 ```
 
 :::warning
-Do not skip ChartsTooltipContainer rendering if the tooltip has no data to display.
-For example the following code does not work.
+If your custom tooltip does not use portal, it can not render inside the ChartContainer.
+Overwise it would render an HTML element inside a SVG.
 
-```jsx
-if (tooltipData === null) {
-  return null;
-}
+The solution is to put your tooltip inside the ChartDataProvider to get data, but outside ChartSurface to avoid being inside the SVG.
```

Comment:
```suggestion
The solution is to render your tooltip as a descendant the ChartDataProvider so it can access the chart data, but outside ChartSurface so it isn't rendered inside an SVG element.
```